.github/workflows/migrate.yml

```diff
@@ -31,18 +31,24 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v3
 
+      - name: Enable Snap parallel-instances
+        run: |
+          sudo snap set system experimental.parallel-instances=true
+
       - name: Setup LXD
         if: matrix.cloud == 'lxd'
         uses: canonical/setup-lxd@ea57509243d3cf39f8ab926e021bb353947b01b5
 
-      - name: Install Juju 3.0
-        run: sudo snap install juju --channel 3.0/stable
+      - name: Install Juju 2.9
+        run: |
+          # TODO(anvial): change to stable channel once juju 2.9.43 is released
```

Comment:
IMHO, we should use stable (after release 2.9.43), because this GHA should test a typical juju user migration path. And "typical" in our situation means: stable.

Reply:
Yeah I agree with @anvial here. If we do later "need to bump the minimum required patch of 2.9", then we can just modify this test.